topic4/contracts/Math.sol

```diff
@@ -3,5 +3,48 @@ pragma solidity ^0.4.23;
 import './interfaces/IMath.sol';
 
 contract Math is IMath {
-		
+
+    uint _stateNumber = 0;
```

Comment:
1. The convention is to prefix the function's input arguments with underscore, in order to differentiate 'em from the state variables. 
2. The default size of uint in solidity is uint256. Choose by yourself which one to use, and be consistent/ use it everywhere. 
3. By default, if you omit the visibility specifier(access modifier), the state variable will be internal. If you mark it public, it will auto-generate getter for you, so the function `stateNumber()` will become redundant. You can read more about visibility and getters here - [link](https://solidity.readthedocs.io/en/develop/contracts.html#visibility-and-getters)